MapleServer2/Types/Levels.cs

```diff
@@ -100,6 +109,7 @@ public void GainExp(int amount)
 
             Exp = newExp;
             Player.Session.Send(ExperiencePacket.ExpUp(amount, newExp, RestExp));
+            DatabaseManager.Update(this);
```

Comment:
We can talk in Discord but this is dicey for the same reason as before. Database calls should not be made on things that will happen this often